internal/yunikorn/rest_test.go

```diff
@@ -773,3 +737,10 @@ func getMockServerYunikornConfig(t *testing.T, serverURL string) *config.Yunikor
 		Port: portNum,
 	}
 }
+
+func writeResponse(w http.ResponseWriter, response any) {
+	err := json.NewEncoder(w).Encode(response)
```

Comment:
Also you can add a `t.Helper()` at the start of the method so reported errors get reported from the parent test case.

Reply:
updated